src/software/run_simulated_ai_vs_ai.sh

```diff
@@ -24,7 +24,7 @@ fi
 
 # download SSL GameController if not downloaded already
 wget -nc https://github.com/RoboCup-SSL/ssl-game-controller/releases/download/v2.4.0/ssl-game-controller_v2.4.0_linux_amd64 -O /tmp/ssl-game-controller_v2.4.0_linux_amd64
-chmod +x /tmp/ssl-game-controller_v2.4.0_linux_amd64
+chmode +x /tmp/ssl-game-controller_v2.4.0_linux_amd64
```

Comment:
is chmode something different or is it supposed to be chmod?

Reply:
nice catch, that's a bad find and replace